webknossos-datastore/app/com/scalableminds/webknossos/datastore/tracings/skeleton/updating/SkeletonUpdateActions.scala

```diff
@@ -11,9 +11,10 @@ import play.api.libs.json._
 
 
 case class CreateTreeSkeletonAction(id: Int, color: Option[com.scalableminds.util.image.Color], name: String,
-                                    branchPoints: List[UpdateActionBranchPoint], timestamp: Long, comments: List[UpdateActionComment], actionTimestamp: Option[Long] = None) extends UpdateAction.SkeletonUpdateAction with SkeletonUpdateActionHelper {
+                                    branchPoints: List[UpdateActionBranchPoint], timestamp: Long, comments: List[UpdateActionComment], actionTimestamp: Option[Long] = None,
+                                    groupId: Option[Int]) extends UpdateAction.SkeletonUpdateAction with SkeletonUpdateActionHelper {
```

Comment:
should `groupID` be `None` as a default? It is in the `Tree` constructor.

Reply:
The Option fields in the update actions are all implicitly None by default, because they are instantiated only by parsing json. (except in your tests now, but I think it’s okay to explicitly fill all values there, if that’s fine with you?)

Reply:
Cool! Sure, that's fine :+1: